v1/ansible/module_utils/facts.py

```diff
@@ -894,6 +894,18 @@ def get_dmi_facts(self):
     @timeout(10)
     def get_mount_facts(self):
         self.facts['mounts'] = []
+        bind_mounts = []
```

Comment:
v1 is dead code, no updates to it are taken, it is there just for being able to quickly test if a bug was preexisting or caused by the v2 engine